uint8 t_adjustRestPose=0
uint8 t_adjustGaitPose=1
uint8 t_startWalking=2
uint8 t_stopWalking=3
uint8 t_spinOnce=4

uint8 gaitControllerCommand

